src/agent/communicator/include/http_socket.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <ihttp_socket.hpp>
+
+#include <boost/asio.hpp>
+#include <boost/asio/experimental/awaitable_operators.hpp>
+#include <boost/beast.hpp>
+#include <boost/system/error_code.hpp>
+
+#include <string>
```

Comment:
Some of these headers could go to the source file.

Reply:
Done